.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,69 @@
+name: Test superset deployment
+
+on:
+  push:
+    branches:
+      - superset-updates # TODO: remove when done testing
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
+
+jobs:
+  test_local:
+    name: Superset local deployment
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          cache: 'npm'
+      - run: sudo apt-get update && sudo apt-get install curl
+      - run: npm ci
+      - run: ./bin/dev --version
+      - run: ./bin/dev config:set registry_host registry.dev.architect.io
+      - run: ./bin/dev config:set api_host https://api.dev.architect.io
+      - run: ./bin/dev config:set log_level debug
+      - run: ./bin/dev login -e ${{ secrets.ARCHITECT_EMAIL }} -p ${{ secrets.ARCHITECT_PASSWORD }}
+      - run: git clone -b superset-updates https://github.com/architect-team/architect-cli.git # TODO: remove this line when this is merged
+      - run: docker volume create volume-key
+      - run: ./bin/dev link examples/hello-world/architect.yml
+      - run: ./bin/dev dev -e superset architect-cli/test/mocks/superset/architect.yml --browser=false -s param_unset=test > ./tmp.txt 2>&1 &
+      - run: sh architect-cli/scripts/server_online.sh
+      - name: Check that the app is accessible locally
+        run: curl --fail -S -I https://hello.localhost.architect.sh
+
+  test_remote:
+    name: Superset remote deployment
+    env:
```

Comment:
Talked offline, but we should set an env here to use in the `architect deploy`. Maybe override hello world text and check the response?